### ISSUE TITLE
Error in contact_list.py sample

### ISSUE THREAD

**Issue by mazulo** [peterbrittain/asciimatics#18]
I was trying test some of samples you provide, but when I try to run `contact_list.py` I get the following:

````bash
└> python contact_list.py
Traceback (most recent call last):
  File "contact_list.py", line 1, in <module>
    from asciimatics.widgets import Frame, ListBox, Layout, Divider, Text, \
ImportError: No module named 'asciimatics.widgets'
````
The same happens if I try from the terminal:
````bash
└> python
Python 3.4.3 (default, Oct 14 2015, 20:28:29) 
[GCC 4.8.4] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from asciimatics.widgets import Frame
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'asciimatics.widgets'
>>> 
````

Am I doing something wrong?

**Comment by peterbrittain** [comment]
Hi @mazulo,

The only thing you're doing wrong is that you're trying to run a sample for the V1.6 release with V1.5 (or earlier) code.  The master branch is for the development latest version of the code (V1.6) that has not yet been published.  I hope to go live soon with the widgets changes that are currently in that branch, but as you'll see in the widget docs, I'm not quite ready yet.

In the meantime, if you want to run this demo, you'll need to clone the master repository and run from that - either by building and installing your own local version of asciimatics or by changing your environment to point directly as the source (e.g. by editing your PYTHONPATH).

Hope that helps...

Peter

**Comment by mazulo** [comment]
I got it! I just upgraded the lib by installing from the `setup.py` of the repo. It works fine :)

I would like to see how it works showing a list of items. I'm building a tool in Python that lists news from a given site. I was using another lib, but it was crashing when the list is greater than 20 items! I was searching for similar libs in PyPI when I found your package.

So I'll try to do this now. Thanks for your help and `asciimatics`!
